backend/.env.e2e.example

```
@@ -0,0 +1,23 @@
+DJANGO_ALLOWED_HOSTS=*
+DJANGO_AWS_ACCESS_KEY_ID=None
+DJANGO_AWS_SECRET_ACCESS_KEY=None
+DJANGO_SETTINGS_MODULE=settings.e2e
+DJANGO_CONFIGURATION=E2E
+DJANGO_DB_HOST=None
+DJANGO_DB_NAME=nest_db_e2e
+DJANGO_DB_USER=nest_user_e2e
+DJANGO_DB_PASSWORD=nest_user_e2e_password
+DJANGO_DB_PORT=5432
+DJANGO_OPEN_AI_SECRET_KEY=None
+DJANGO_PUBLIC_IP_ADDRESS="127.0.0.1"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)